### PR TITLE
#92 - AnonymizeCommand - Make output more compact in none-verbose mode

### PR DESCRIPTION
**Pull request by SimonMellerin**

### CODE REVIEW

**Comment by SimonMellerin** [comment]
Here is the new compact version:

image

File: src/Command/Anonymization/AnonymizeCommand.php [hunk 1]
```diff
@@ -237,7 +239,11 @@ protected function execute(InputInterface $input, OutputInterface $output): int
 
     private function doBackupInitialDatabase(): void
     {
-        $this->io->section('Start backuping local database');
+        if ($this->io->isVerbose()) {
+            $this->io->section('Start backing up local database');
```
Comment (Lonnytunes) [inline src/Command/Anonymization/AnonymizeCommand.php:244]: `"` instead of `'` to keep consistent with the rest? :slightly_smiling_face:

Comment (SimonMellerin) [reply src/Command/Anonymization/AnonymizeCommand.php:244]: I made the opposit :)

**Comment by SimonMellerin** [comment]
new version:

image

File: src/Command/Anonymization/AnonymizeCommand.php [hunk 2]
```diff
@@ -218,14 +218,17 @@ protected function execute(InputInterface $input, OutputInterface $output): int
             }
 
             $this->doAnonymizeDatabase();
-
-            if (!$this->doAnonymizeCurrentDatabase && !$this->doBackupAndRestoreInitial) {
-                $this->doBackupAnonymizedDatabase();
-            }
+            $this->doBackupAnonymizedDatabase();
```
Comment (SimonMellerin) [inline src/Command/Anonymization/AnonymizeCommand.php:221]: I add a fix: conditions were wrong here, following what's said in command description, anonymized database should always be backed up.

Comment (SimonMellerin) [reply src/Command/Anonymization/AnonymizeCommand.php:221]: The description:

>  Anonymize a given backup file or the local database.
> 
>                 This command will successively perform these steps:
>                     1/ Backup the local database,
>                     2/ Restore the given backup file,
>                     3/ Anonymize the data from the given backup file,
>                     4/ Backup the newly anonymized database by overwritting the given backup file,
>                     5/ Restore your database to its original state from the backup produced at step 1.
> 
>                 If called with the --local-database option, step 2 is skipped.
>                 If called with the --no-restore option, step 1 and 4 are skipped.

**Review by Lonnytunes — APPROVED** [review]

**Pull request merged**